cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControlConfiguration.java

```diff
@@ -30,7 +30,9 @@
                 CruiseControl.DISK_CAPACITY_GOAL,
                 CruiseControl.NETWORK_INBOUND_CAPACITY_GOAL,
                 CruiseControl.NETWORK_OUTBOUND_CAPACITY_GOAL,
-                CruiseControl.CPU_CAPACITY_GOAL,
+                // TODO: The CPU metric are currently not reported correctly when running on Kubernetes
+                //       we should add this back in once fixed upstream
+                //CruiseControl.CPU_CAPACITY_GOAL,
```

Comment:
if we have an upstream issue I would add the link here as well, otherwise we should open one and linking it here?

Reply:
I added a link to the issue @kyguy made in the upstream repo